ush/create_model_configure_file.py

```diff
@@ -297,8 +296,11 @@ def parse_args(argv):
 if __name__ == "__main__":
     args = parse_args(sys.argv[1:])
     cfg = get_yaml_config(args.path_to_defns)
-    cfg = flatten_dict(cfg)
-    import_vars(dictionary=cfg)
+    import_vars(dictionary={
+        **cfg["task_run_fcst"],
+        **cfg["workflow"],
+        }
+        )
```

Comment:
Cutting down this nonsense to use only the variables from a smaller subset of the whole config as Python locals.